documentation.dd

```diff
@@ -0,0 +1,6 @@
+Ddoc
+
+$(D_S Documentation, $(NAVIGATION_DOCUMENTATION))
+
+Macros:
+    TITLE=Documentation
```

Comment:
That is, it's a fallback page for when there's no JS. community.dd, resources.dd, search.dd, and menu.dd are fallback pages, too.